app/assets/javascripts/success_prospects.es6

```diff
@@ -0,0 +1,17 @@
+$(function() {
+  $('#success-prospect-text-area').hide();
+  var option_name = $("input[type=radio][name='merits_assessment[success_prospect]']:checked").val()
```

Comment:
As this files uses ES6, I think `var` should be replace with `let`

Reply:
changed